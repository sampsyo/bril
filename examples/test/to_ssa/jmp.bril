@main() {
.entry:
    cond: bool = const true;
    # No `phi` needed here since there is no branching.
    b: int = const 42;
    jmp .exit;
.exit:
    b: int = const 0;
    print b;
}
