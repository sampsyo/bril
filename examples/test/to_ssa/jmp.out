@main {
.entry:
  cond.0: bool = const true;
  b.0: int = const 42;
  jmp .exit;
.exit:
  b.1: int = const 0;
  print b.1;
  ret;
}
